operations/enterprises.md

```diff
@@ -217,6 +217,87 @@ Returns all departments of an enterprise associated with the connector integrati
 | `IsActive` | boolean | required | Whether the department is still active. |
 | `Name` | string | required | Name of the department. |
 
+## Get all counters
+
+Returns all counters of an enterprise associated with the connector integration.
```

Comment:
are they associated with integration anyhow ?

Reply:
No, but the enterprise is. We have this format in other places as well.